src/manage_migrators.c

```diff
@@ -1737,6 +1737,60 @@ migrate_208_to_209 ()
   return 0;
 }
 
+/**
+ * @brief Migrate the database from version 208 to version 209.
+ *
+ * @return 0 success, -1 error.
+ */
+int
+migrate_209_to_210 ()
+{
+  sql_begin_immediate ();
+
+  /* Ensure that the database is currently version 209. */
+
+  if (manage_db_version () != 209)
+    {
+      sql_rollback ();
+      return -1;
+    }
+
+  /* Update the database. */
+
+  /* Remove the fields "bid" and "xref" from table "nvts".
+   * This is done by dropping the entire table and
+   * unset the sync time stamps to force gvmd to refresh the content. */
+
+  sql ("DROP TABLE IF EXISTS nvts CASCADE;");
+
+  sql ("DELETE FROM meta where name = 'nvts_feed_version' OR name = 'nvts_check_time';");
+
+  sql ("CREATE TABLE IF NOT EXISTS nvts"
```

Comment:
This is not necessary.  Manager will create the table on startup.  We only need
to add a CREATE TABLE when we're modifying a table that may not have
been created.

Reply:
I tried it when I developed it and it did not work, the existence of the table was required.

Reply:
It should work.  But perhaps you ran into some dependency problem
that we don't usually see because the tables are usually created
together.

Would prefer ALTER TABLE, because we usually do it that way, and
only create the table if it is needed by the migrator.

Reply:
It is changed to apply ALTER TABLE now.